src/Spryker/Client/RabbitMq/Model/Connection/QueueConnectionTransferMapper/QueueConnectionTransferMapper.php

```diff
@@ -0,0 +1,133 @@
+<?php
+
+/**
+ * MIT License
+ * For full license information, please view the LICENSE file that was distributed with this source code.
+ */
+
+namespace Spryker\Client\RabbitMq\Model\Connection\QueueConnectionTransferMapper;
+
+use Spryker\Client\RabbitMq\Model\Exception\ConnectionConfigIsNotDefinedException;
+use Spryker\Client\RabbitMq\RabbitMqConfig;
+
+class QueueConnectionTransferMapper implements QueueConnectionTransferMapperInterface
+{
+    protected const CONNECTION_CONFIG_IS_NOT_DEFINED_EXCEPTION_MESSAGE = 'Couldn\'t find configuration "%s" to create connection. Please, check `RabbitMqEnv::RABBITMQ_CONNECTIONS` in `config_*.php` files, or your `stores.php`';
```

Comment:
Not a blocker, but usually I would prefer to have constant name like `CONSTANT_PURPOSE_CONSTANT_NAME`. E.g. `EXCEPTION_MESSAGE_CONNECTION_CONFIG_IS_NOT_DEFINED`